plugin.video.hrmagplayer/addon.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.hrmagplayer" name="HR Magazine Player" version="1.0.0" provider-name="extramundane">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+		<license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+		<source></source>
+        <summary lang="en">Live streaming and magazines from Hessischer Rundfunk TV</summary>
+        <description lang="en">This add-on provides access to the HR TV live stream and some local cultural shows.</description>
+        <assets>
```

Comment:
please define the language element which reflects the language of the content (videos) the addon provides. See https://kodi.wiki/view/Addon.xml#.3Clanguage.3E